docs/source/filtering.rst

```diff
@@ -0,0 +1,75 @@
+.. _filtering:
+
+~~~~~~~~~~~~~~~~~~~~~~~~
+Filtering vector indexes
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+CuVS supports different type of filtering depending on the vector index being used. The main method used in all of the vector indexes
```

Comment:
Nitpick: Please write cuVS with lowercase 'c', even at the beginning of a sentence.